src/modules/rlm_radutmp/rlm_radutmp.c

```diff
@@ -267,7 +267,7 @@ static unlang_action_t CC_HINT(nonnull) mod_accounting(rlm_rcode_t *p_result, mo
 			ut.nas_port = vp->vp_uint32;
 			port_seen = true;
 		} else if (vp->da == attr_acct_delay_time) {
-			ut.delay = vp->vp_uint32;
+			ut.delay = fr_time_delta_to_sec(vp->vp_time_delta);
```

Comment:
this is correct, as `delay` is seconds, and `vp->vp_time_delta` is `fr_time_delta_t`.